data/eFLOWER_Dun_2022/metadata.yml

```diff
@@ -0,0 +1,1609 @@
+    (fascicles) in one whorl and was recorded as a continuous character (with integer
+    values of 1 and above). We call this character structural because both fertile
+    stamens and staminodes were considered. This character is not applicable for spiral
+    or irregular stamen arrangements, nor when there is a single structural stamen
+    (i.e., one stamen, no staminodes; e.g., Chloranthus, Chloranthaceae).
+- var_in: 406. Fusion of styles (C1)
+  unit_in: '{dimensionless}'
+  trait_name: flower_style_fusion
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: Here we record the degree of fusion of styles at anthesis, recorded on
+    a continuous scale, from 0 (free styles) to 1 (styles fused along their entire
+    length, but excluding the stigmatic region). Partial fusion is recorded using
+    an approximate number between these two extremes (e.g., 0.1 corresponds to basal
+    fusion, 0.5 to fusion along half of the total length of the styles). This character
+    is not applicable in unicarpellate flowers.
+- var_in: 406. Fusion of styles (C1)_min
+  unit_in: '{dimensionless}'
+  trait_name: flower_style_fusion
+  entity_type: species
+  value_type: minimum
+  basis_of_value: measurement
+  replicates: .na
+  methods: Here we record the degree of fusion of styles at anthesis, recorded on
+    a continuous scale, from 0 (free styles) to 1 (styles fused along their entire
+    length, but excluding the stigmatic region). Partial fusion is recorded using
+    an approximate number between these two extremes (e.g., 0.1 corresponds to basal
+    fusion, 0.5 to fusion along half of the total length of the styles). This character
+    is not applicable in unicarpellate flowers.
+- var_in: 406. Fusion of styles (C1)_max
+  unit_in: '{dimensionless}'
+  trait_name: flower_style_fusion
+  entity_type: species
+  value_type: maximum
+  basis_of_value: measurement
+  replicates: .na
+  methods: Here we record the degree of fusion of styles at anthesis, recorded on
+    a continuous scale, from 0 (free styles) to 1 (styles fused along their entire
+    length, but excluding the stigmatic region). Partial fusion is recorded using
+    an approximate number between these two extremes (e.g., 0.1 corresponds to basal
+    fusion, 0.5 to fusion along half of the total length of the styles). This character
+    is not applicable in unicarpellate flowers.
+- var_in: 120. Floral orientation (D1)
+  unit_in: .na
+  trait_name: flower_orientation
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: unknown
+- var_in: 250. Perianth colour (D1)
+  unit_in: .na
+  trait_name: perianth_colour
+  entity_type: species
+  value_type: mode
+  basis_of_value: expert_score
+  replicates: .na
+  methods: The dominant colour of the perianth parts (sepals, petals, and tepals).
+- var_in: 150. Floral colour (D1)
+  unit_in: .na
+  trait_name: flower_colour
```

Comment:
I noticed from this dataset report that the other dataset with flower_colour values (NHNSW_2014_2) has a lot of repeated values (e.g. 'yellow_orange yellow_orange', 'green white_cream white_cream'). Assuming this happened when separate colours were collapsed into the allowable values that combined two colours.

Reply:
Yes, and those were collapsed in an automated manner - so I'm going to leave them for now.